Source/Details/ASTraitCollection.m

```diff
@@ -32,6 +32,14 @@ ASDISPLAYNODE_INLINE UIContentSizeCategory AS_UIContentSizeCategoryUnspecified()
   }
 }
 
+ASDISPLAYNODE_INLINE ASPrimitiveContentSizeCategory safePrimitiveContentSizeCategory(ASPrimitiveContentSizeCategory sizeCategory) {
+  return sizeCategory ? sizeCategory : AS_UIContentSizeCategoryUnspecified();
+}
```

Comment:
I think in both cases we have to use `ASPrimitiveContentSizeCategoryMake(AS_UIContentSizeCategoryUnspecified())`